compiler/src/dmd/declaration.d

```diff
@@ -542,6 +542,11 @@ extern (C++) abstract class Declaration : Dsymbol
         return (storage_class & STC.future) != 0;
     }
 
+    final extern(D) bool isSystem() const pure nothrow @nogc @safe
```

Comment:
Please update the header

Reply:
It's `extern(D)`, do you think it should be exposed to the `extern(C++)` interface?

Reply:
Oh, I missed that. Thats a question for @ibuclaw or @kinke

Reply:
Nope, `extern(D)` does not need to be exposed to other backends.